test/unit/extract/csv_test.py

```diff
@@ -70,7 +70,7 @@ def test_extract():
             # Testing the rename
             GenericMetadata,
             "get_column_map",
-            return_value={company_field: "company_rename"},
+            return_value={"company_rename": company_field},
```

Comment:
I'm confused about what's going on with the swaparoo in this test.

Reply:
ah yes. the swaparoo needed to happen in this test because it was in the wrong order and it was making the unit tests fail bc this test actually runs an extraction validation which now has the column map failure.